datastructure/graphlist.py

```diff
@@ -0,0 +1,45 @@
+class Vertex:
+    def __init__(self, name):
```

Comment:
is name its value?

Reply:
Yes. Should I change it to `value`? 
But I think it should stay as `name`, since `value` could be mistaken as the weight of the edge.

Reply:
If you want to future proof it, i think each node should have an id, a value (string for now), and edges.

weights (or costs) are typically for edges, since it usually is something like a penalty to travel down an edge (like a road may be longer)

Reply:
ID? What do I put in the ID?